source/lib-intro.tex

```diff
@@ -1295,7 +1284,7 @@
 \ref{support.rtti}  & Type identification          &   \tcode{<typeinfo>}      \\ \rowsep
 \ref{support.exception} & Exception handling      &   \tcode{<exception>}     \\ \rowsep
 \ref{support.initlist}  & Initializer lists & \tcode{<initializer_list>} \\ \rowsep
-\ref{support.runtime} & Other runtime support     &   \tcode{<cstdalign>} \tcode{<cstdarg>} \tcode{<cstdbool>}       \\ \rowsep
```

Comment:
I "fixed" this problem by reverting the changes to Clause 17. I noticed that we already list `<strstream>` as part of the library, even though we only ever mention it again in Annex D. We can do the same with these four headers.